### DISCUSSION TITLE
Monthly ESMValTool community meeting April 2023

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#97] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next online community meeting will be in April. Note that there will be also an in-person meeting at EGU, see #90 for details.

The result of the poll is that the meeting will take place on

Tuesday, April 18 at 1 PM CEST

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

**Format and agenda**

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #84

**Comment by bouweandela** [comment]
Updates
- @egalytska Submitted a paper with ESMValTool and working on pull request. It was mentioned that it was not easy to find the [code contribution guidelines](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#).
- @glpotter Asked if there is a tutorial on how to use ESMValTool to CMORize reanalysis data. @bouweandela recommends opening an issue at https://github.com/esMValGroup/esmvaltool_tutorial/issues to discuss this with the @ESMValGroup/userengagementteam 
- @valeriupredoi Is working on Python 3.11 support ESMValGroup/ESMValCore#1776

**Reply by remi-kazeroni** [reply]
- @valeriupredoi working on enabling support for Python3.11.
- @FranziskaWinterstein using ESMValTool for model evaluation, including EMAC. 
- @remi-kazeroni did the release of v2.8 last month together with @valeriupredoi.

**Comment by remi-kazeroni** [comment]
Reminder: the next opportunity to meet people from the ESMValTool community will be at EGU 2023 next week (24-28 April). For more information, see [EGU 2023 informal and splinter meeting](https://github.com/ESMValGroup/Community/discussions/90)
